airbyte-cdk/bulk/toolkits/load-iceberg-parquet/src/main/kotlin/io/airbyte/cdk/load/data/iceberg/parquet/IcebergParquetPipelineFactory.kt

```diff
@@ -20,10 +22,10 @@ class IcebergParquetPipelineFactory : MapperPipelineFactory {
         MapperPipeline(
             stream.schema,
             listOf(
-                // TODO not sure why base parquet was doing this as a noop
-                SchemalessTypesToStringType() to SchemalessValuesToJsonString(),
-                AirbyteSchemaNoopMapper() to NullOutOfRangeIntegers(),
+                SchemalessTypesToStringType to SchemalessValuesToJsonString(),
```

Comment:
open thread - should iceberg also include the "fail on unknown type" mapper that avro/parquet have https://airbytehq-team.slack.com/archives/C03AS1GAQV6/p1734968675676689?thread_ts=1734747025.083309&cid=C03AS1GAQV6

(this is causing the test failure)

Reply:
Rrrrright. It would make the most sense to fail on Unknown in the coerce mapper, since the point of that is to validate.

But we can't because Avro is using Unknown as a sentinel type for NullType.

But... if we had an explicit NullType... 🙃